packages/contracts/build/contracts/IdentityEvents.json

```diff
@@ -876,7 +876,11 @@
     "name": "solc",
     "version": "0.4.25+commit.59dbf8f1.Emscripten.clang"
   },
-  "networks": {},
+  "networks": {
+    "999": {
+      "address": "0xeec918d74c746167564401103096D45BbD494B74"
```

Comment:
I think we can revert the changes to the contract build files as these addresses won't get used anywhere

Reply:
Ugh. Yes, this should not have been committed.